modules/form-field/react/stories/stories_Radio.tsx

```diff
@@ -11,7 +11,7 @@ import README from '../../../radio/react/README.md';
 const hintText = 'Helpful text goes here.';
 const hintId = 'error-desc-id';
 
-storiesOf('Form Field/Radio/Top Label/Radio Group', module)
+storiesOf('Components|Inputs/Radio/Top Label/Radio Group', module)
```

Comment:
Do you think anyone will have a hard time finding these stories? They're no longer categorized under form-field in storybook and the readme in storybook shows the checkbox module. I think the organization is in the right place, but it's a concern. Not sure of the best way to solve this 🤔

Reply:
I think it's ok for now. The most important thing is alignment with design side of the house...If we get feedback on category name/find-ability we can always update.

Reply:
A valid concern @anicholls, this is something that always is going to happen until certain degree when introducing a new ordering but as @sahlhoff said with the new change we are getting alignment and from now one will makes sense, since we have a clear structure and naming convention in place. I don't think this is something we will be changing very often anyways 😊